src/misc/emptyValue.ts

```diff
@@ -0,0 +1,21 @@
+import { setEmpty as dSetEmpty } from "../datetime/setEmpty";
+import { setEmpty as sSetEmpty } from "../text/setEmpty";
+import { setEmpty as nSetEmpty } from "../numeric/setEmpty";
+import { setEmpty as bSetEmpty } from "../bool/setEmpty";
+
+/**
+ * @param value
+ * @return any
+ */
+export const emptyValue = (value: any): any => {
```

Comment:
We could be more specific here too:

```suggestion
export const emptyValue = (value: boolean | number | string | Date | null | undefined): any => {
```